cpp/src/ray/runtime/task/task_executor.cc

```diff
@@ -71,18 +84,19 @@ std::unique_ptr<ObjectID> TaskExecutor::Execute(InvocationSpec &invocation) {
 };
 
 std::pair<Status, std::shared_ptr<msgpack::sbuffer>> GetExecuteResult(
-    const std::string &lib_name, const std::string &func_name,
-    const std::vector<msgpack::sbuffer> &args_buffer, msgpack::sbuffer *actor_ptr) {
-  auto entry_func = FunctionHelper::GetInstance().GetEntryFunction(lib_name);
-  if (entry_func == nullptr) {
-    return std::make_pair(ray::Status::NotFound(lib_name + " not found"), nullptr);
+    const std::string &func_name, const std::vector<msgpack::sbuffer> &args_buffer,
+    msgpack::sbuffer *actor_ptr) {
+  try {
+    auto funcs = FunctionHelper::GetInstance().GetExecutableFunctions(
```

Comment:
As commend at the function GetExecutableFunctions, here we can explicit call the get GetExecutableFunction and GetExecutableMemberFunction would be very clear and easy to read.